static/js/public/store-categories.js

```diff
@@ -0,0 +1,47 @@
+import Carousel from "./carousel";
+
+/**
+ *
+ * @param {HTMLElement} holder
+ * @returns {Promise<string>}
+ */
+function getCategory(holder) {
+  const category = holder.dataset.category;
+
+  // Write the html and init the carousel
+  const writeCategory = function(response) {
+    holder.innerHTML = response;
+    new Carousel(holder.querySelector('.p-carousel'));
+  };
+
+  const url = `/store/categories/${category}`;
```

Comment:
This looks like an URL to a webpage, not internal API. If we start having APIs, we should consider having `/api` URL namespace for it or something.

Reply:
I agree. Originally I did it as an 'API' like endpoint, but because I'm passing HTML through I didn't want to label it as 'API'. We will definitely want to make it a better API in the future